config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+  resources :recipe_foods
+  resources :foods
+  resources :recipes
```

Comment:
- [ ] Kindly restrict your routes to only allow actions in use, the requirements say update should not be used.

![crud](https://github.com/ikoote1/Recipe-App/assets/53124548/d0c49c83-2d1d-4c37-8179-5de762601afc)